### ISSUE TITLE
problem of adv loss and performance

### ISSUE THREAD

**Issue by YufanZZZ** [Gandolfczjh/3D2Fool#21]
<img width="1076" alt="Image" src="https://github.com/user-attachments/assets/99b270e5-95d9-417b-8a50-3daf72184405" />

Hi author, I tried to reproduce the result with the default parameter, but the loss of adv is hard to converge. I run 10 epochs and found the result is not very good. 

I'd like to ask what possible reasons there are.

Thanks!

<img width="1502" alt="Image" src="https://github.com/user-attachments/assets/4b7108b4-6ace-47f1-a49c-8213e678df93" />

**Comment by Gandolfczjh** [comment]
You can try adjusting the coefficients before the loss terms, as different victim models may require different settings, which need to be manually tuned. Therefore, the coefficients here are not strictly consistent with those in the paper. We have already provided the best attack results for the MonoDepth2 model—you can test the attack effectiveness using the texture seed we provided.

**Comment by YufanZZZ** [comment]
Another question: How to use your texture seed in the code, because I only see the texture_seed.png file and orginal camou_para = torch.rand([1, h, w, 3]).float().to(args.device).

**Comment by Gandolfczjh** [comment]
Here, camou_para1 refers to the PNG file I provided. You can use the Pillow library to read the image and convert it into a PyTorch tensor.

```python
# continuous color
camou_para = torch.rand([1, h, w, 3]).float().to(args.device)
camou_para.requires_grad_(True)
optimizer = optim.Adam([camou_para], lr=args.lr)
camou_para1 = expand_kernel(camou_para.permute(0, 3, 1, 2)).permute(0, 2, 3, 1)
```

**Comment by leige521** [comment]
@YufanZZZ Hi, can I add your contact details? I would like to ask a question about how to render the patch to the car model of the CARLA simulator, and I am willing to pay some money.

**Comment by Ian-Tam** [comment]
@YufanZZZ Hi, my results using the default parameters are quite similar to yours. Did you manage to find the optimal parameters?
